diesel/src/sqlite/connection/raw.rs

```diff
@@ -115,8 +129,12 @@ impl RawConnection {
         num_args: usize,
     ) -> QueryResult<()>
     where
-        A: SqliteAggregateFunction<Args, Output = Ret> + 'static + Send,
-        Args: FromSqlRow<ArgsSqlType, Sqlite>,
+        A: SqliteAggregateFunction<Args, Output = Ret>
+            + 'static
+            + Send
+            + std::panic::UnwindSafe
```

Comment:
I've changed it to both `RefUnwindSafe` and `UnwindSafe`, since `::step()` passes by reference and `::terminate()` by value. As [stated before](https://github.com/diesel-rs/diesel/pull/2495#discussion_r487028556), I'm not sure if the `RefUnwindSafe`-bound for `::step()` provides any practical safety since we just use `AssertUnwindSafe` instead to pass the `mut&` across and `RefUnwindSafe` is only for non-`mut` references.